disk-utils/fdisk-menu.c

```diff
@@ -111,6 +111,7 @@ static const struct menu menu_generic = {
 		MENU_ENT  ('t', N_("change a partition type")),
 		MENU_BENT_E('v', N_("verify the partition table"), FDISK_DISKLABEL_BSD),
 		MENU_ENT  ('i', N_("print information about a partition")),
+		MENU_ENT  ('e', N_("resize a partition")),
```

Comment:
Better ideas for the menu character are very welcome.
'r' is already used unfortunately.